config/configModule.ts

```diff
@@ -0,0 +1,17 @@
+let env = process.env.NODE_ENV || 'development'
+let localConfig: any
+
+try {
+  localConfig = require('./config.json')
```

Comment:
Can we require that for HollowBot to work a path to `hollowbot.config.json` must exist and that's where it reads the values?